README.md

```diff
@@ -53,6 +53,7 @@ Perspectives of the role of MiNiFi should be from the perspective of the agent a
   * ListenSyslog
   * ExecuteProcess
   * AppendHostInfo
+  * ProvenanceTaskReport
```

Comment:
We should make this SiteToSiteProvenanceReportingTask for consistency between instance types.